### DISCUSSION TITLE
Set 'selected' item in Select tag

### DISCUSSION THREAD

**Discussion by beaker1977** [vmware-clarity/ng-clarity#711] [General]
I'm loading a 'select' with data from a model and I'd like to set one of the option items as 'selected', say 'warning' as pre-selected in the list. How can I do so? Trying to set the selected property with a value in the model doesn't work. Is there even a way to do so?

<select clrSelect name="options" [(ngModel)]="opts" required>
     <option *ngFor="let opt of opts" [value]="opt .type" name="type" selected="{{opt .selected}}">{{opt .text}}</option>
</select>

options = [
    {
      type: 'danger',
      text: 'Danger',
      selected: '',
    },
    {
      type: 'warning',
      text: 'Warning',
      selected: 'selected',
    },
    {
      type: 'info',
      text: 'Info',
      selected: '',
    },
  ];

**Comment by kevinbuhmann** [comment]
This question is not specific to Clarity. Set the `ngModel` to the selected option value. You're currently setting it to an array, and that's not how `ngModel` works...
